### PR TITLE
#2604 ensure score sent to LMS before status

### PR DESCRIPTION
**Pull request by chris-steele**
Fixes https://github.com/adaptlearning/adapt_framework/issues/2604

See https://github.com/adaptlearning/adapt-contrib-spoor/pull/222

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/tracking.js [hunk 1]
```diff
@@ -36,9 +36,21 @@ class Tracking extends Backbone.Controller {
...
 
+  submitScore() {
+    if (!this._config._shouldSubmitScore) return;
+  
+    if (this._assessmentState.isPercentageBased) {
+      Adapt.offlineStorage.set('score', this._assessmentState.scoreAsPercent, 0, 100);
+      return;
+    }
+  
+    Adapt.offlineStorage.set('score', this._assessmentState.score, 0, this._assessmentState.maxScore);
```
Comment (oliverfoster) [inline js/tracking.js:51]:
```suggestion
    Adapt.offlineStorage.set('score', this._assessmentState.score, this._assessmentState.minScore, this._assessmentState.maxScore);
```
Assessment state also has a `minScore` now https://github.com/adaptlearning/adapt-contrib-assessment/pull/132

**Review by oliverfoster — CHANGES_REQUESTED** [review]

**Review by oliverfoster — APPROVED** [review]
pending version bump

**Comment by oliverfoster** [comment]
Sorry Chris, this probably should be a minor version bump. So 6.1.0 as it's a feature addition really rather than a bugfix. Fine line in this case.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**